src/v2/Apps/Article/Components/ArticleAd/ArticleAd.tsx

```diff
@@ -51,6 +51,8 @@ ArticleAd.displayName = "ArticleAd"
 
 export const ArticleAdProvider: FC = ({ children }) => {
   return (
-    <DFPSlotsProvider dfpNetworkId="21805539690">{children}</DFPSlotsProvider>
+    <DFPSlotsProvider dfpNetworkId="21805539690" lazyLoad>
```

Comment:
What is DFPS?

Reply:
"Double Click for Publishers" — it's now known as GPT ("Google Publisher Tag")